content/influxdb/clustered/install/set-up-cluster/test-cluster.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Test your InfluxDB Cluster
+description: >
+  Test to ensure your InfluxDB cluster can write and query data successfully.
+menu:
+  influxdb_clustered:
+    name: Test your cluster
+    parent: Set up your cluster
+weight: 250
+---
+
+With your InfluxDB cluster deployed and running, test to ensure you can
+successfully write and query data from InfluxDB.
+
+1. [Download and install influxctl](#download-and-install-influxctl)
+1. [Retrieve your cluster's admin token](#retrieve-your-clusters-admin-token)
+1. [Configure influxctl to connect to your cluster](#configure-influxctl-to-connect-to-your-cluster)
+1. [Create a new database](#create-a-new-database)
+1. [Write test data to the new database](#write-test-data-to-the-new-database)
+1. [Query the test data from your database](#query-the-test-data-from-your-database)
+
+## Download and install influxctl
+
+[`influxctl`](/influxdb/clustered/reference/cli/influxctl/) is a command line tool
+that lets you manage, write data to, and query data from your InfluxDB cluster
+from your local machine.
+
+<a href="/influxdb/clustered/reference/cli/influxctl/#download-and-install-influxctl" class="btn" target="_blank">Download and install influxctl</a>
+
+## Retrieve your cluster's admin token
+
+InfluxDB Clustered generates a valid access token (known as the _admin token_)
+and stores it as a secret in your cluster's `influxdb` namespace.
+During this phase of the installation process, use the admin token with
+`influxctl` in lieu of configuring and using an identity provider.
+
+Use `kubectl` to retrieve the admin token from your `influxdb` namespace secret
+store and copy it to a file:
+
+```sh
+kubectl get secrets/admin-token \
+  --template={{.data.token}} \
+  --namespace influxdb | base64 -d > token.json
+```
+
+## Configure influxctl to connect to your cluster
+
+Create an [`influxctl` connection profile](/influxdb/clustered/reference/cli/influxctl/#configure-connection-profiles)
+for your InfluxDB cluster. Connection profiles are stored in a `config.toml`
+file on your local machine and contain the credentials necessary to connect to
+and authorize with your InfluxDB cluster.
+
+1.  Create a file named `config.toml` with the following contents:
+
+    {{< code-placeholders "INFLUXDB_(HOST|PORT)|DIRECTORY_PATH" >}}
+```toml
+[[profile]]
+  name = "default"
+  product = "clustered"
+  host = "{{< influxdb/host >}}"
```

Comment:
In the staging docs environment this shows up as `host = "undefined"`. Will that be fixed when these changes are deployed to production? Regardless, I think we should reference ingress controller setup here (`influxdb/clustered/install/set-up-cluster/prerequisites/#set-up-a-kubernetes-ingress-controller`) since the ingress controller LB IP is what they will need to use here.

Reply:
> In the staging docs environment this shows up as `host = "undefined"`. Will that be fixed when these changes are deployed to production?

Hmm, ok. The "undefined" is a cookie issue. There should be a script that populates a cookie with all the default host values. Can you do me a favor and run the following in your browser console and let me know what the output is:

```javascript
getUrls()
```

> Regardless, I think we should reference ingress controller setup here (`influxdb/clustered/install/set-up-cluster/prerequisites/#set-up-a-kubernetes-ingress-controller`) since the ingress controller LB IP is what they will need to use here.

Ok, good to know. Most of our docs work on the assumption that users know the entry point/host of their InfluxDB instance/cluster, but I can specifically call this out here though. Should we update the `cluster-host` placeholder to `load-balancer-host`/`load-balancer-ip`? What would be most familiar to the end-user here?